bentoml/cli/utils.py

```diff
@@ -126,6 +155,14 @@ def _format_deployment_age_for_print(deployment_pb):
         return humanfriendly_age_from_datetime(deployment_pb.created_at.ToDatetime())
 
 
+def humanfriendly_bytes(num):
+    step_unit = 1000  # apparently docker uses 1000?
+    for x in ['B', 'KB', 'MB', 'GB', 'TB']:
+        if num < step_unit:
+            return "%3.2f %s" % (num, x)
+        num /= step_unit
```

Comment:
You can try use the package humanfriendly. It is already part of the required modules for BentoML.

https://github.com/xolox/python-humanfriendly#a-note-about-size-units